_specification-v2/components.md

```diff
@@ -0,0 +1,137 @@
+set the `origamiType` property in the JSON to `"component"`.
+
+## Naming conventions
+
+A component’s name is used in the package.json, URLs, and CSS class names, so:
+
+- It **must** contain only ASCII letters, numbers and hyphens
+- It **must** begin with a letter
+
+<aside>
+	Examples of good component names include
+	<code>o-colors</code>,
+	<code>o-grid</code> and
+	<code>o-cookie-message</code>.
+</aside>
+
+## Package management
+
+Origami components **must** be installable with version 7 of npm package
+manager.
+
+As well as following the
+[package.json](https://docs.npmjs.com/cli/v7/configuring-npm/package-json) spec,
+there are additional requirements to make the component's npm manifest conform
+to the Origami specification.
+
+- It **must** include a
+[`"browser"`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#browser)`
+  property set to the component's main JavaScript entry-point if the component
+  has JavaScript.
+- It **must not** include a
+  [`"browser"`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#browser)
+  property if the component has no JavaScript.
+- It **must** include a `"sass"` property set to the component's main Sass
+  entry-point if the component has Sass.
+- It **must not** include a `"sass"` property if the component has no Sass.
+- It **must** contain a `"description"` property set to a short description of the
+  component.
+- It **must** contain a `"keywords"` property in order to help users discover the right
+  component.
+- It **must** include a `"type"` property set to `"module"`.
+- It **must** include a `"license"` property set to the SPDX license identifier for the
+  license of the component, e.g. "MIT".
+- It **must** list any production Sass or Origami component dependencies under
+  `"peerDependencies"`.
+- It **must** include the bugs field as described in the package.json spec. The
+  bugs field may contain a `"slack"` property, indicating the support channel on
+  the FT’s internal Slack workspace.
+
+## Documentation
+
+A component **must** contain a README.md.
+
+When a new major version of a component is released a migration guide **must** be
+written to help users upgrade from the previous release. The migration guide
+**must** be added to a MIGRATION.md file in the root of the component's codebase,
+and **must** be linked to from the component's README.md.
+
+<aside>
+	See
+	<a href="https://github.com/Financial-Times/o-table#migration">
+		o-table's migration guide
+	</a>
+	as an example.
+</aside>
+
+## HTML
+
+- A component’s markup must be contained in a root element that must:
+	- Have a data-o-component attribute with value of the component’s name, or
+	- Be defined as a
+	  [customElement](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-element)
+	  registered against the component’s class
+- All data attributes and IDs must be namespaced with the component’s name.
```

Comment:
Adding the boldness on the `must`s, do we need to add boldness the the `or` at the end of the first bullet point?
```suggestion
- A component’s markup **must** be contained in a root element that **must**:
	- Have a data-o-component attribute with value of the component’s name, or
	- Be defined as a
	  [customElement](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-element)
	  registered against the component’s class
- All data attributes and IDs **must** be namespaced with the component’s name.
```